component/sqs/component.go

```diff
@@ -90,29 +91,38 @@ type retry struct {
 }
 
 type config struct {
-	maxMessages       *int64
-	pollWaitSeconds   *int64
-	visibilityTimeout *int64
+	maxMessages       int32
+	pollWaitSeconds   int32
+	visibilityTimeout int32
 }
 
 type stats struct {
 	interval time.Duration
 }
 
+type API interface {
+	CreateQueue(ctx context.Context, params *sqs.CreateQueueInput, optFns ...func(*sqs.Options)) (*sqs.CreateQueueOutput, error)
```

Comment:
Same here is the component interested in all these methods? We should focus only on the methods that the component needs.

Reply:
Updated, PTAL.